packages/core/src/sdk/analytics/platform/vtex/search.ts

```diff
@@ -156,6 +165,18 @@ const handleEvent = (
       break
     }
 
+    case 'intelligent_search_autocomplete_click': {
+      sendEvent({
+        text: event.params.term,
+        url: event.params.url ?? '',
+        productId: event.params.productId ?? '',
```

Comment:
question: Why not add it conditionally like you did for `position`?

Reply:
Because this one is required, `position` isn't.

Reply:
Got it! Did you check with the Search team to confirm it's ok to send empty string?

Reply:
Yes, we confirmed that for the autocompleted terms the `productId` and `position` are not required, we'll send as an empty string just to bypass the Events API requirement. I've tried to send without the `productId`, but it throws an error.